fiftyone/operators/loader.py

```diff
@@ -157,6 +162,15 @@ def exec_module_from_dir(module_dir, plugin_definition):
         )
     spec = importlib.util.spec_from_file_location(mod_dir, mod_filepath)
     mod = importlib.util.module_from_spec(spec)
+    try:
+        pctx = register_module(plugin_definition, mod)
```

Comment:
We don't want to call `register_module()` twice.

Reply:
yeah I forgot to remove the previous call. Fixed now